.github/workflows/ccruntime_e2e_amd.yaml

```diff
@@ -0,0 +1,51 @@
+name: ccruntime e2e tests amd sev
+on:
+  workflow_call:
+
+permissions:
+  contents: read
+
+jobs:
+  e2e:
+    name: operator tests
+    strategy:
+      fail-fast: false
+      matrix:
+        runtimeclass: [kata-qemu-sev, kata-qemu-snp]
+        instance: [amd-coco-ci-ubuntu2004-001, amd-milan-coco-ci-ubuntu2004-001]
+        exclude:
+          - runtimeclass: kata-qemu-snp
+            instance: amd-coco-ci-ubuntu2004-001
+          - runtimeclass: "kata-qemu-sev"
+            instance: "amd-milan-coco-ci-ubuntu2004-001"
+    runs-on: [self-hosted]
```

Comment:
label value goes here

Reply:
Adi expalined this to me. Closing comment.

Reply:
Hmm, would you care to elaborate a bit? Ideally in a form of a comment inside the workflow as the same questions arise and will arise in the future by people coming to the team?